conans/test/integration/command/install/install_test.py

```diff
@@ -447,3 +447,18 @@ def test_create_cli_override(self, client):
                      "test_package/conanfile.py": GenConanfile().with_test("pass")})
         client.run("create . pkg/0.1@ --require-override=zlib/2.0")
         assert "zlib/2.0: Already installed" in client.out
+
+
+def test_install_bintray_warning():
+    """
+    IMPORTANT: This test is actually using https://conan.bintray.com
+    Warning is only displayed when downloading a recipe from the remote
+    """
+    client = TestClient()
+    client.run("remote add conan-center https://conan.bintray.com")
```

Comment:
I tried to avoid using the real bintray remote with a test server using the same URL. However, this lead to modifications to the TestServer class and others in the requester...

Anyway, I think it could be useful to have a test that tracks the availability of the remote so we can drop the message once it is no longer available 🤷‍♂️

Reply:
Then, in my opinion, I think we should use a "fake" Bintray URL and check the output:

```python
    client = TestClient()
    fake_bintray_url = "https://myfakebintray.com"
    with patch.object(proxy, "BINTRAY_DEPRECATED_URL", return_value=fake_bintray_url):
         client.run("remote add conan-center %s" % fake_bintray_url)
         # .....
```
What do you think @danimtb?